modules/sqlline/pom.xml

```diff
@@ -39,12 +39,7 @@
         <dependency>
             <groupId>sqlline</groupId>
             <artifactId>sqlline</artifactId>
-            <version>1.3.0</version>
-        </dependency>
-        <dependency>
-            <groupId>jline</groupId>
```

Comment:
Sergey, why we are removing `jline` dependency?

Reply:
jline is included in the sqlline dependency